planetiler-basemap/src/main/java/com/onthegomap/planetiler/basemap/layers/Place.java

```diff
@@ -340,7 +337,7 @@ public void process(Tables.OsmCityPoint element, FeatureCollector features) {
 
     int minzoom = rank != null && rank == 1 ? 2 :
       rank != null && rank <= 8 ? Math.max(3, rank - 1) :
-      placeType.ordinal() <= PlaceType.TOWN.ordinal() ? 7 :
+      placeType.ordinal() <= PlaceType.TOWN.ordinal() ? 4 :
```

Comment:
Can you add a link from this PR's description or from a commit message in this PR back to the PR or commit in the openmaptiles repo which made the same label change?

Reply:
https://github.com/ACMEAtronOmatic/openmaptiles/pull/1/commits/6bb701b7b26055238552c2553c453625366b47bd

Reply:
Also: https://github.com/ACMEAtronOmatic/openmaptiles/pull/1